### ISSUE TITLE
Supplying own AndroidSDKPath does not handle missing '/'

### ISSUE THREAD

**Issue by nhaarman** [linkedin/qark#5]
When supplying a pre-installed Android SDK directory without the trailing `/`, Qark does not handle this correctly:

```
Do you want QARK to download and set up Android SDK?[y/n] :n
Please enter the fullpath to your Android SDK root directory:/opt/android-sdk
Updated config value:: AndroidSDKPath /opt/android-sdk
INFO - Initializing QARK
```
For example, when creating and installing the explotation APK, the following happens:

```
Do you want to install this to your device? (y/n)y
Traceback (most recent call last):
  File "qark.py", line 1041, in <module>
    apkList = list_all_apk()
  File "qark.py", line 319, in list_all_apk
    st = os.stat(adb)
OSError: [Errno 2] No such file or directory: '/opt/android-sdkplatform-tools/adb'
```

**Comment by SecBro1** [comment]
Thanks for letting us know. We'll get it patched shortly.

**Comment by tushardalvi** [comment]
We believe this issue has been resolved with the v0.9-alpha.10 release. Please take a look at it and let us know if you are still seeing the issue.
